src/presentation/http/router/note.test.ts

```diff
@@ -77,7 +77,40 @@ describe('Note API', () => {
       expect(response?.json()).toStrictEqual(expectedNote);
     });
 
-    test('Returns 403 when public access is disabled, user is not creator of the note', async () => {
+    test('Returns note by public id with 200 status when access is disabled, but user is creator', async () => {
+      const expectedStatus = 200;
+      const userId = 1;
+      const accessToken = global.auth(userId);
+
+      const expectedNote = {
+        'id': 3,
+        'publicId': '73NdxFZ4k7',
+        'creatorId': 1,
+        'content': null,
+        'createdAt': '2023-10-16T13:49:19.000Z',
+        'updatedAt': '2023-10-16T13:49:19.000Z',
+      };
+
+      const notPublicNote = notes.find(newNote => {
```

Comment:
name of the variable does not fully match the login inside .find(). I suggest either renaming or adding a comment

Reply:
> name of the variable does not fully match the login inside .find(). I suggest either renaming or adding a comment

done